graphql.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+require('dotenv').config({
+    path: path.resolve(process.cwd(), 'packages/venia-concept/.env')
+});
+
+const excludePatterns =
+    process.env.MAGENTO_BACKEND_EDITION === 'EE'
+        ? ['!**/*.ce.js']
+        : ['!**/*.ee.js'];
+
+const config = {
+    schema: [
+        `${process.env.MAGENTO_BACKEND_URL}/graphql`,
+        '**/client-schema.graphql'
+    ],
+    documents: [
+        './packages/{peregrine,venia-ui,venia-concept}/{lib,src}/**/*.{js,graphql,gql}',
+        ...excludePatterns
+    ],
+    extensions: {
+        codegen: {
+            generates: {
+                './merged-schema.v4.json': {
```

Comment:
`v4`?

Reply:
New config structure for `graphql-config@4`. This codegen is actually broken until we fix a lot of other naming collisions, so I may remove this until we can resolve those.

Reply:
Removed codegen section in 52ed88d